packages/11ty/content/_assets/styles/components/quire-copyright.scss

```
@@ -17,12 +17,12 @@
     & img {
       max-height: 100%;
     }
-    &__logo {
-      height: 3em;
-      margin-bottom: .25em;
-      margin-right: 1em;
-      max-width: 100%;
-      display: inline-block;
-    }
+  }
+  .copyright__publisher-logo {
+    height: 3em;
+    margin-bottom: .25em;
+    margin-right: 1em;
+    max-width: 100%;
+    display: inline-block;
   }
```

reorder style class properties
```suggestion
  .copyright__publisher-logo {
    display: inline-block;
    height: 3em;
    margin-bottom: 0.25em;
    margin-right: 1em;
    max-width: 100%;
  }
```